### PR TITLE
Clarify Quick Ingest review estimates

### PR DESCRIPTION
**Pull request by rmusser01**
## Summary
- Fix Quick Ingest Review copy so estimate approximation markers are rendered once by `formatEstimate()`.
- Add a regression test covering duplicate markers in the review summary and long-duration warning.
- Close `TASK-394.2` with verification notes.

## Test Plan
- [x] `./node_modules/.bin/vitest run src/components/Common/QuickIngest/__tests__/QuickIngestWizardModal.integration.test.tsx --maxWorkers=1 --no-file-parallelism`
- [x] `./node_modules/.bin/vitest run src/components/Layouts/__tests__/QuickIngestButton.resume.test.tsx --maxWorkers=1 --no-file-parallelism`
- [x] `git diff --check`

## Browser Smoke
- [x] Started Next dev server at `http://127.0.0.1:18082` with local test API-key env and verified `/media` renders the Quick Ingest launcher.
- [ ] Modal interaction was not completed because an unrelated dev runtime overlay from `GET /api/v1/media?page=1&results_per_page=20&include_keywords=true` blocked the page.

## Change Summary
TODO: Human requester must replace this with a human-written summary before merge per `Docs/superpowers/AI_GENERATED_PR_CHANGE_SUMMARY_POLICY_2026_04_17.md`.

---
## Summary by cubic
Clarified Quick Ingest review estimates. Centralized the "~" in `formatEstimate()` and updated i18next and extension locales so the summary and long-time warning show a single marker.

- **Bug Fixes**
  - Removed "~" from review templates and added `review.summary` and `review.warnLongTime` locale entries without it across both locale bundles.
  - Strengthened the integration test to cover all time units and catch duplicate markers.
  - Satisfies TASK-394.2 copy clarity for the Review step.

<sup>Written for commit ad968ab88833dde3402189679447a3cfab37d628. Summary will update on new commits.</sup>

<a href="https://cubic.dev/pr/rmusser01/tldw_server/pull/2092?utm_source=github">Review in cubic</a>

### CODE REVIEW

File: apps/packages/ui/src/components/Common/QuickIngest/ReviewStep.tsx [hunk 1]
```diff
@@ -166,7 +166,7 @@ export const ReviewStep: React.FC<ReviewStepProps> = ({
     // Long estimated time
     if (totalEstimatedSeconds > LONG_TIME_THRESHOLD) {
       result.push(
-        qi("review.warnLongTime", "Processing may take a while (~{{time}})", {
+        qi("review.warnLongTime", "Processing may take a while ({{time}})", {
           time: estimatedTimeLabel,
         })
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Common/QuickIngest/ReviewStep.tsx:171]: Fixed in ad968ab88: added explicit Quick Ingest locale entries for `quickIngest.review.warnLongTime` and `quickIngest.review.summary` in `assets/locale/en/option.json` plus the extension public locale bundles, all without a leading `~`. Verified the locale JSON/static check has no `~{{time}}` templates.

File: backlog/tasks/task-394.2 - Clarify-Quick-Ingest-entry-and-destination-copy.md [hunk 2]
```diff
@@ -24,17 +25,39 @@ Execute implementation plan Task 2: improve first-time clarity, destination expe
...
+- [x] #3 Accessible labels and keyboard/focus behavior are preserved or improved
 <!-- AC:END -->
 
+## Implementation Notes
+
+<!-- SECTION:NOTES:BEGIN -->
+- Started from merged dev 66c8bec4d in isolated worktree .worktrees/notes-post-merge-next. Scope: execute plan Task 2 only, focused Quick Ingest launcher/Add step copy and tests.
+
+- Merged dev already included the first-open Add step purpose/destination copy and the Quick Ingest launcher wording/aria-label baseline.
+- Added a regression for Review step estimate copy to prevent duplicate approximation markers (for example, `~~21 min estimated`).
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply backlog/tasks/task-394.2 - Clarify-Quick-Ingest-entry-and-destination-copy.md]: Fixed in ad968ab88: changed the Backlog note to say “regression test” so it clearly refers to test coverage rather than an introduced defect.

**Comment by rmusser01** [comment]
Addressed current review feedback in ad968ab88:

- Qodo: estimate-copy assertions now accept `sec`, `min`, or `hr` while still asserting no duplicate `~~` marker.
- Gemini: added explicit `quickIngest.review.warnLongTime` and `quickIngest.review.summary` locale entries in the i18next and extension locale bundles without a leading `~`.
- CodeRabbit: clarified the Backlog note wording from “regression” to “regression test”.

Verification:
- `./node_modules/.bin/vitest run src/components/Common/QuickIngest/__tests__/QuickIngestWizardModal.integration.test.tsx --maxWorkers=1 --no-file-parallelism`
- `./node_modules/.bin/vitest run src/components/Layouts/__tests__/QuickIngestButton.resume.test.tsx --maxWorkers=1 --no-file-parallelism`
- locale JSON/static check for no `~{{time}}` templates
- `git diff --check`

All inline review threads are replied to and resolved. CI is still pending. The PR description still has the required human-written Change Summary placeholder before merge.
